src/client/components/features/Accompagnement/DemandeDeContact/Formulaire/FormulaireDemandeDeContactAccompagnement.tsx

```diff
@@ -129,11 +130,18 @@ export function FormulaireDemandeDeContactAccompagnement({
 			/>
 			{isLoading
 				? <LoadingButton className={styles.formulaireValidateButton}/>
-				: <ButtonComponent
-					type="submit"
-					className={styles.formulaireValidateButton}
-					label="Envoyer mes informations afin d‘être rappelé(e)"
-				/>
+				: <>
+					<ButtonComponent
+						type="submit"
+						className={classNames(styles.formulaireValidateButton, styles.desktopOnly)}
+						label="Envoyer mes informations afin d‘être rappelé(e)"
+					/>
+					<ButtonComponent
+						type="submit"
+						className={classNames(styles.formulaireValidateButton, styles.mobileOnly)}
+						label="Envoyer mes informations"
+					/>
+				</>
```

Comment:
Je me demande si ça serait pas plus simple de faire un truc du genre
```suggestion
				: <>
					<ButtonComponent
						type="submit"
						className={classNames(styles.formulaireValidateButton, styles.desktopOnly)}
						label=<>Envoyer mes informations <span className="desktopOnly">afin d‘être rappelé(e)</span></>
					/>
				</>
```
Il faudrait juste changer le typage de `label: string` en `label: ReactNode`

Reply:
bonne idée !